Source/ZXing.Net.Mobile.Android/ZXingSurfaceView.cs

```diff
@@ -166,8 +166,29 @@ void OnSurfaceChanged ()
 
             var parameters = camera.GetParameters ();
             parameters.PreviewFormat = ImageFormatType.Nv21;
-            parameters.SetPreviewFpsRange(15, 30);
-            
+
+            // First try continuous video, then auto focus, then fixed
+            var supportedFocusModes = parameters.SupportedFocusModes;
+            if (supportedFocusModes.Contains (Camera.Parameters.FocusModeAuto))
+                parameters.FocusMode = Camera.Parameters.FocusModeAuto;
+            else if (supportedFocusModes.Contains (Camera.Parameters.FocusModeContinuousVideo))
+                parameters.FocusMode = Camera.Parameters.FocusModeContinuousVideo;
+            else if (supportedFocusModes.Contains (Camera.Parameters.FocusModeFixed))
+                parameters.FocusMode = Camera.Parameters.FocusModeFixed;
+
```

Comment:
Yep I have this fixed in another branch that will be merged soon. A bit messy currently.